lib/modules/userTagger.js

```diff
@@ -811,9 +811,15 @@ export async function applyTagToAuthor(thisAuthorObj: HTMLAnchorElement, noEdit:
 		);
 	}
 
+	let tagElement;
+	if (module.options.showTaggingIcon.value) {
+		tagElement = addAuthorTag(thisAuthorObj, noEdit, thisAuthor);
+	}
+
 	const tagObject = await getTagBatched(thisAuthor);
 
-	if (module.options.showTaggingIcon.value || !_.isEmpty(tagObject)) {
+	if (!_.isEmpty(tagObject)) {
+		if (tagElement && tagElement.parentElement) tagElement.parentElement.removeChild(tagElement);
```

Comment:
Is that using the jQuery definition or predefined? I try to avoid jQuery if I can since it's not included in the new profile build, but if you'd prefer jQuery extensions, I'll just use those.